Client/Dockerfile

```diff
@@ -1,20 +1,15 @@
 FROM gradle:7.4.2-jdk8 AS build
 COPY  . /home/gradle/src
 WORKDIR /home/gradle/src
-ARG settings=./settings.xml
-RUN mkdir ~/.m2
-COPY $settings ~/.m2/settings.xml
-RUN gradle build 
+ENV GITHUB_USERNAME=[GITHUB_USERNAME]
```

Comment:
Try this answer to make the Dockerfile more user-friendly [https://stackoverflow.com/questions/69321584/environment-variables-inside-dockerfile](https://stackoverflow.com/questions/69321584/environment-variables-inside-dockerfile)

Reply:
Yes I see. The reason I didn't have it passed as arguements was because then the command line for running the Dockerfile would become a bit too long and tedious. Do you think I should still proceed as such?

Reply:
Yeah I think having those tokens as CLI arguments is better